src/Password/Bcrypt.php

```diff
@@ -19,7 +18,7 @@
  */
 class Bcrypt implements PasswordInterface
 {
-    const MIN_SALT_SIZE = 16;
+    const MIN_SALT_SIZE = 22;
```

Comment:
Example 3 reported in the manual, [here the link](http://php.net/manual/en/function.password-hash.php#refsect1-function.password-hash-examples).